IyiKuyoro/Pals
# #161533651: setup testing

#### What does this PR do?
Modify Development Dependencies

#### How should this be manually tested?
>- Clone this repository
>- Open the test directory and add tests to the test.js file
>- Use npm to check for the status of the tests

#### What are the relevant Pivotal Tracker stories?

#161533651
